code/game/gamemodes/miniantags/abduction/machinery/experiment.dm

```diff
@@ -216,6 +216,35 @@
 	occupant = null
 	update_icon(UPDATE_ICON_STATE)
 
+/obj/machinery/abductor/experiment/relaymove()
+	if(!occupant)
+		return
+	to_chat(occupant, "<span class='warning'>You start trying to break free!</span>")
+	if(!do_after(occupant, 30 SECONDS, FALSE, src))
```

Comment:
This needs to be toned down a *lot*. As it currently is, first you have to wake up from the abductor baton, then you have to spend an average of 1.5 m trying to break free. If you're unlucky enough to get sedated and don't succeed the next attempt, you're gonna get put to sleep for 30 seconds or so. If you're extra unlucky and get sedated twice in a row, you'll be out for about 2 minutes instead. It's all of the randomness of Genetics, on an even slower time scale, and with nothing else you can do at the same time. Not fun.

If you want to have randomized effects, the timer should be more like 5 seconds, and you definitely shouldn't administer ether as one of the possibilities. Otherwise, 30 seconds would be fine for unconditional escape.